src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -29,24 +27,92 @@ internal override bool ShouldPersistOption(OptionKey key)
 
         #region "Preview Text"
 
-        private static readonly string s_declarationPreviewTrue = @"
+        private static readonly string s_fieldDeclarationPreviewTrue = @"
+class C{
+    int capacity;
+    void foo()
```

Comment:
Let's avoid using `foo` in option descriptions.  Also, I'd follow style guidelines for these.  How about `Method()`?